core/controllers/access_validators.py

```diff
@@ -78,6 +78,29 @@ def get(self) -> None:
             raise self.PageNotFoundException
 
 
+class CollectionPageAccessValidationHandler(
```

Comment:
Can we rename this to CollectionPlayerPageAccessValidationHandler/CollectionViewerPageAccessValidationHandler? Because we also have collection editor page in the queue for Angular migration.

Reply:
Okay